.azure/templates/steps/system_test_general.yaml

```diff
@@ -149,6 +149,7 @@ jobs:
         RESOURCE_ALLOCATION_STRATEGY: "NOT_SHARED"
         TEST_LOG_DIR: $(test_log_dir)
         CONNECT_IMAGE_WITH_FILE_SINK_PLUGIN: $(connectImage)
+        KAFKA_TIERED_STORAGE_BASE_IMAGE: '$(docker_registry)/$(docker_org)/kafka:$(docker_tag)-kafka-$(st_kafka_version)'
```

Comment:
What if the `st_kafka_version` is not set?

Reply:
Yeah good point, I wanted to try if it will work that way and forgot about the `latest` Kafka version. Thanks